app/main/views.py

```diff
@@ -16,11 +19,16 @@ def index():
     req_opt_desc = Descriptor.query.filter_by(
         id=req_opt_desc.descriptor_id
     ).first()
+    options = Descriptor.query.all()
+    options = [o for o in options if len(o.text_resources) == 0 and o.id is not req_opt_desc.descriptor_id]
```

Comment:
What do you mean by 1? I mean that sometimes a required option descriptor wont be set so we cant just call the descriptor_id as if it will always be there and have to set it to some dummy that will always evaluate to true like -1 since no id will be negative